app/models/sitemap_indexer.rb

```diff
@@ -12,28 +14,30 @@ def initialize(sitemap_url:)
   end
 
   def index
-    sitemaps.any? ? enqueue_sitemaps : process_entries
+    enqueue_sitemaps
+    process_entries
   end
 
   private
 
-  def sitemaps
-    @sitemaps ||= Sitemaps.parse(sitemap).sitemaps
-  end
-
-  def sitemap_entries
-    # Eventually we might add an option to the Sitemaps gem to limit the URLS
-    # to those strictly adhering to the sitemap protocol, but this should suffice for now
-    # https://www.pivotaltracker.com/story/show/157485118
```

Comment:
Please move this comment above the section where we ensure that the domain of the URL matches the domain of the sitemap:
```
if URI(entry['loc'].strip).host == domain
```